lib/http.dart

```diff
@@ -28,7 +28,8 @@ export 'src/response.dart';
 /// the request is complete. If you're planning on making multiple requests to
 /// the same server, you should use a single [Client] for all of those requests.
 ///
-/// For more fine-grained control over the request, use [Request] instead.
+/// For more fine-grained control over the request and response, create a
+/// [Client] and use it directly.
```

Comment:
Now that these methods support streaming requests and responses, I don't think we need these notes anymore.